roles-in-tech/developer-advocate.md

```diff
@@ -44,13 +41,16 @@ Steps towards a better Developer Evangelist:
 
```

Comment:
Can you remove the quotes here and add the name of the slideshow?

Reply:
@alodahl  do you mean the name of the book? there is no slideshow. Shall I mention the book name and add a link to it?

Reply:
["Link to the SlideShare:"](https://docs